install/kubernetes/helm/subcharts/security/templates/enable-mesh-mtls.yaml

```diff
@@ -35,7 +35,7 @@ spec:
       mode: ISTIO_MUTUAL
 ---
 # Destination rule to disable (m)TLS when talking to API server, as API server doesn't have sidecar.
-# Customer should add similar destination rules for other services that dont' have sidecar.
+# Customer should add similar destination rules for other services that don't' have sidecar.
```

Comment:
don't' -> don't

Reply:
It is fixed, please review again @geeknoid